tutorials/review.adoc

```diff
@@ -0,0 +1,881 @@
+>> Start next lesson
+
+==== Index Terms
+
+Before you define an index term, your document needs a section where the index terms can appear. To create an index, define a level 1 section (==) marked with the style index at the end of your document.
+
+[source, AsciiDoc]
+----
+[index]
+== Index
+----
+Index entries can consist of up to three levels using a comma to separate the terms. Metanorma provides two options to create an index entry:
+
+* Visible index terms are words within the text that are annotated with an index entry using double parentheses. `\((Level 1 index term))`
+* Hidden index terms are index entries that are not visible in the final output and are generated using triple parentheses. `(\((Level 1 index term, Level 2 index term, Level 3 index term)))`
+
+Let’s have a look at an example:
+
+[source, AsciiDoc]
+----
+The Lady of the Lake, her arm clad in the purest shimmering samite,
+held aloft Excalibur from the bosom of the water,
+signifying by divine providence that I, ((Arthur)), <1>
+was to carry Excalibur (((Sword, Broadsword, Excalibur))). <2>
+----
+<1> `\((Arthur))` will be displayed as `Arthur` in the text and carries a first level index entry
+<2> `(\((Sword, Broadsword, Excalibur)))` will not appear in the text, but a three level index entry will be generated.
+
+[[exercise-2-10]]
+include::exercises.adoc[tag=exercise-2-10]
+
+In the next lesson, we will cover references and links. 
+>> Start next lesson
+
+==== References
+
+References are a core part of standards, because they coexist and relate to other standard documents. There are four types of references:
+
+* Hyperlinks to an external source, for example a link to a website
+* Metadata references
+* Internal references to a section, image, table, etc.
+* Bibliographic entries
+
+*Hyperlinks*
+
+AsciiDoc lets you include links to external sources by pasting the URL into the document and appending the link text in square brackets `[Link text]`. 
+
+.Example of a link
+[source, AsciiDoc]
+----
+http://www.iso.org/[International Organization for Standardization].
+----
+
+*Metadata references*
+
+Every document contains a set of metadata to describe the document. You can insert a metadata reference by putting the attribute in curly braces `{attribute}`. The reference will be replaced with the value in the rendered output. 
+
+[source, AsciiDoc]
+----
+This document was prepared by Technical Committee ISO/TC {technical-committee-number}, _{technical-committee}_, Subcommittee SC {subcommittee-number}, _{subcommittee}_.
+----
+
+
+*Internal references*
+
+To link to an important place in your document, you need to set an anchor using double square brackets `[[anchor]]`. Metanorma automatically numbers and names these references. Since the markup will be converted into XML, the anchor must follow the https://www.w3.org/TR/xml-names11/[XML namespace conventions]. Therefore, an anchor name name must not contain: 
```

Comment:
```suggestion
To link to an important place in your document, you need to set an anchor using double square brackets `[[anchor]]`. Metanorma automatically numbers and names these references; because they are autonumbered, they will be renumbered automatically if you insert any new text of the same type. Since the markup will be converted into XML, the anchor must follow the https://www.w3.org/TR/xml-names11/[XML namespace conventions]. Therefore, an anchor name name must not contain: 
```